tests/utils/frameworks/tensorflow_utils.py

```diff
@@ -12,7 +12,8 @@ def __init__(self, units=32, **kwargs):
         super(CustomLayer, self).__init__(**kwargs)
         self.units = tf.Variable(units, name="units")
 
-    def call(self, inputs, training=False, **kwargs):
+    # for review
+    def call(self, inputs, training=False):
```

Comment:
Are the `kwargs` for call compatibility? Should I add a pylint ignore here instead?

Reply:
it should be ok to remove `**kwargs` here